rasa/core/policies/ted_policy.py

```diff
@@ -332,12 +338,15 @@ def get_default_config() -> Dict[Text, Any]:
             # ingredients in a recipe, but it doesn't make sense for the parts of
             # an address
             SPLIT_ENTITIES_BY_COMMA: SPLIT_ENTITIES_BY_COMMA_DEFAULT_VALUE,
+            # Max history of the policy, unbounded by default
+            POLICY_MAX_HISTORY: DEFAULT_MAX_HISTORY,
+            # Determines the importance of policies, higher values take precedence
+            POLICY_PRIORITY: DEFAULT_POLICY_PRIORITY,
         }
 
-    @staticmethod
-    def _standard_featurizer(max_history: Optional[int] = None) -> TrackerFeaturizer:
+    def _standard_featurizer(self) -> TrackerFeaturizer:
         return MaxHistoryTrackerFeaturizer(
-            SingleStateFeaturizer(), max_history=max_history
+            SingleStateFeaturizer(), max_history=self.config.get(POLICY_MAX_HISTORY)
```

Comment:
we can even do this, no?
```suggestion
            SingleStateFeaturizer(), max_history=self.config[POLICY_MAX_HISTORY]
```